File: Python/test_temp.py
import unittest
from AOC import *

class Test(unittest.TestCase):
    def test_part1(self):
        file = open("test_input.txt", "r")
        input = file.read().splitlines()
        file.close()
        expected = open("test_expected.txt", "r").read().splitlines()
        self.assertTrue(False)

    def test_part2(self):
        file = open("test_input.txt", "r")
        input = file.read().splitlines()
        file.close()
        expected  = open("test_expected.txt", "r").read().splitlines()
        self.assertTrue(False)
        
if __name__ == "__main__":
    unittest.main()
    

File: Go/test_temp.go
package main

import (
	"fmt"
	"os"
	"testing"
)

func TestPart1(t *testing.T) {
	input, err := os.ReadFile("test_input.txt")
	if err != nil {
		panic(err)
	}

	fmt.Println(input)
	if true == false {
		t.Errorf("Test Not Implemented")
	}
}

func TestPart2(t *testing.T) {
	input, err := os.ReadFile("test_input.txt")
	if err != nil {
		panic(err)
	}

	fmt.Println(input)
	if true == false {
		t.Errorf("Test Not Implemented")
	}
}
